src/Organizer/Events/GeoCoordinatesUpdated.php

```diff
@@ -5,8 +5,9 @@
 use CultuurNet\Geocoding\Coordinate\Coordinates;
 use CultuurNet\Geocoding\Coordinate\Latitude;
 use CultuurNet\Geocoding\Coordinate\Longitude;
+use CultuurNet\UDB3\JsonLdSerializableInterface;
 
-class GeoCoordinatesUpdated extends OrganizerEvent
+class GeoCoordinatesUpdated extends OrganizerEvent implements JsonLdSerializableInterface
```

Comment:
I know this is an existing interface and it might seem to make sense to use this, but I'd rather not for new code because it ends up mixing the write model (events) with the read model (json-ld). In the long run I'd rather use https://github.com/cultuurnet/udb3-models to build the JSON-LD documents, but that's not in scope of this ticket. Can you instead just set the `geo` property in the `OrganizerLDProjector` without this interface & `toJsonLd`? Just doing something like 
```
$this->geo = [
    'latitude' => $geoCoordinatesUpdated->getCoordinates()->getLatitude()->toDouble(),
    'longitude' => $geoCoordinatesUpdated->getCoordinates()->getLongitude()->toDouble(),
];
```

is fine for now.

Reply:
Done